### ISSUE TITLE
Change permissions between tests

### ISSUE THREAD

**Issue by tuzmusic** [kamranayub/cypress-browser-permissions#44]
As far as I can tell from the docs, this allows you to set the permissions before your suite runs.
I need the ability to set different permissions for different tests, to test the effect of those different permissions. 
Does this plugin allow for this?

**Comment by savchukm** [comment]
I'm not sure but looks like it doesn't allow to do that. Global browserPermissions in `cypress.config.js` take over any test suite permission:
```javascript
export default defineConfig({
	e2e: {
	        ...,
		env: {
			browserPermissions: {
				geolocation: 'allow',
			}
		},
	...
})
```
```javascript
it(`It should work`, {
	env: {
		browserPermissions: {
			geolocation: 'block',
		}
	}
}, () => {
    console.log(Cypress.env()) // here geolocation is 'block' but actual permission is always 'allow' for all tests
})
})
```

Cypress 10.7.0
@kamranayub What am I missing here?

**Comment by kamranayub** [comment]
I don't think that will work because `launchOptions` are only set when launching the browser. You can't modify launch options after it's been launched. 

Instead, you could try passing the `CYPRESS_` env override variable values in a separate test script you run separately. Use a different test suite and skip those suites in the other script.

For testing permissions _specifically_ you don't _need_ to disable them 😃 You can mock them.

See this test suite (which has permissions allowed by default at launch, but I test that when they are blocked, the UI still works):

https://github.com/kamranayub/example-carved-rock-fitness-order-tracker/blob/master/e2e/cypress/integration/notifications.spec.js
